layouts/partials/header.html

```diff
@@ -35,6 +35,7 @@
       {{ end }}
     </style>
     {{ partial "custom-header.html" . }}
+    {{ partial "mathjax_support.html" . }}
```

Comment:
You're missing this file in your commit :-(

Reply:
OK I first added it in the projects using hugo-theme-learn but it's easier to put it here.